### PR TITLE
Fix some typos in brand names and use Oxford list for an enumeration

### PR DESCRIPTION
**Pull request by ccoVeille**
I'm using your tool and I noticed a few "typos"

So I'm suggesting a few changes

### CODE REVIEW

File: readme/install.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ The executable will be installed to `/usr/bin`.
 
 ## macOS
 
-### With [Homebrew](https://brew.sh/)
+### With [HomeBrew](https://brew.sh/)
```
Comment (MitMaro) [inline readme/install.md:95]: I don't believe that it's HomeBrew, everywhere on https://brew.sh/ is Homebrew.

File: readme/customization.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ The valid colors are the [eight original 8 ANSI colors][ANSIColors]. They are `b
 
 ## Key Bindings
 
-Most keys can be changed to any printable character or supported special character. It is possible to provide conflicting bindings, which will result in undefined behaviour. The `inputConfirmYes` binding has a special behaviour in that it responds to both the uppercase and lowercase letter of the value set, if the variant exist.
+Most keys can be changed to any printable character or supported special character. It is possible to provide conflicting bindings, which will result in undefined behavior. The `inputConfirmYes` binding has a special behavior in that it responds to both the uppercase and lowercase letter of the value set, if the variant exist.
```
Comment (MitMaro) [inline readme/customization.md:104]: This is technically the Canadian (and UK) way of spelling behaviour, however, I've already used the American spelling of Colour, so this makes sense to change.

**Review by MitMaro — CHANGES_REQUESTED** [review]
Thanks for the suggestions and fixes!

HomeBrew is the only thing that shouldn't be changed, if you don't mind changing that one back?

File: readme/install.md (hunk 1 above)
Comment (ccoVeille) [reply readme/install.md:95]: You are right. I will fix tomorrow. I'm not in front of a computer now.

File: readme/customization.md (hunk 2 above)
Comment (ccoVeille) [reply readme/customization.md:104]: I know, this one is open to debate.

I would say the English way would be colour or behaviour.

But that said, in code software, the US English often defines the norm.

I mean CSS refers to color, not colour.

I let you tell me if you want me to restore the behavior, I don't mind as I already have to fix Homebrew

Comment (MitMaro) [reply readme/customization.md:104]: I agree, changing to the US spelling makes sense here!

**Review by MitMaro — APPROVED** [review]

**Comment by MitMaro** [comment]
Thanks for the various fixes!

**Pull request merged**